README.md

```diff
@@ -0,0 +1,12 @@
+#Native login docs
+
+This repo is about writing the docs for the new Native Login system.
```

Comment:
1) The repo is not 'about writing the docs' I believe... Maybe it 'contains the docs'?
2) NativeLogin - without a whitespace in between.